emacs-everywhere.el

```diff
@@ -353,6 +374,7 @@ Never paste content when ABORT is non-nil."
   "Return information on the active window."
   (let ((w (pcase system-type
              (`darwin (emacs-everywhere-app-info-osx))
+	     ((or `ms-dos `windows-nt `cygwin) (emacs-everywhere-app-info-windows))
```

Comment:
Indentation is wrong.

Reply:
Gave `(emacs-everywhere-app-info-windows)` its new line.